packages/RatPack.package/RPTests.class/instance/testPutObjectOnApplication.st

```diff
@@ -0,0 +1,16 @@
+test - REST operations
+testPutObjectOnApplication
+	
+	| url urlId putResult postResult testJson otherTestJson|
+	app registerModel: RPTestModel.
+	url := app class apiRoutePrefix, (RPTestModel name).
+	testJson := RPTestHelper getExampleJson.
+	otherTestJson := RPTestHelper getOtherExampleJson.
+	self assert: testJson ~= otherTestJson.
+	urlId := url, '/1'.
+	postResult := RPTestHelper post: url on: app content: testJson.
+	putResult := RPTestHelper put: urlId on: app content: otherTestJson.
+		
+	self assert: ((postResult contents contents) ~= (putResult contents contents)).
+	self assert: ((putResult contents contents) includesSubString: otherTestJson).
```

Comment:
What functionality are you trying to test here? Asserting that the responses differ but one includes the other seems not so straight forward for me. What do you think about just getting the put result and comparing it like in the test above? We're already testing post there, so we don't need to do that again. (Edit: That's exactly what the test below does. I don't quite understand what we're testing here.)